### DISCUSSION TITLE
Welcome to Comment.nvim Discussions!

### DISCUSSION THREAD

**Discussion by numToStr** [numToStr/Comment.nvim#198] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

**Comment by npquintos** [comment]
It is nice for comments to be nearly invisible (as it is right now), but when reviewing code (when on normal mode), it would be nice to make them very visible (to make them easier to read) that could be triggered with the cursor being on them. This should happen on all adjacent comments, including those that are separated by newlines, but is otherwise adjacent. Is it possible to add this feature? Thanks.
